src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js

```diff
@@ -44,9 +44,10 @@ const ReportsDailyVisits = ({ domainName, dateFrom, dependencies: { datahubClien
   useEffect(() => {
     const fetchData = async () => {
       setState({ loading: true });
-      const dailyVisitsData = await datahubClient.getDailyVisitsByPeriod({
+      const dailyVisitsData = await datahubClient.getDailyAndHoursVisitsByPeriod({
         domainName: domainName,
         dateFrom: dateFrom,
+        periodBy: 'days',
```

Comment:
I would prefer `groupBy` or `groupByPeriod` but DataHub API already accept the parameter `periodBy`, so, it seems that this one is the best name.

Reply:
yes i saw Datahub api use this thats why use that